src/index.html

```diff
@@ -8,6 +8,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="star"></div>
```

Comment:
```suggestion
      <div class="stars__star"></div>
```
![Screenshot from 2023-05-04 13-21-44](https://user-images.githubusercontent.com/33029864/236177708-5c1c0179-f3a4-4ca3-a204-c2b79e466165.png)

Reply:
![image](https://user-images.githubusercontent.com/117664824/236178563-9d7ec1af-4714-4df7-8635-4ffa8a90661a.png)